src/translations/de-DE.json

```diff
@@ -11,6 +11,7 @@
     "visualizations.totals.dropdown.title.min": "Min",
     "visualizations.totals.dropdown.title.max": "Max",
     "visualizations.totals.dropdown.title.nat": "Rollup (Gesamt)",
+    "visualizations.totals.dropdown.title.nat.disabled.tooltip": "Rollup (Gesamt) Aggregation ist nicht verfügbar, wenn die Betrachtung nach Messwert gefiltert wird. Um Rollup (Gesamt) zu verwenden, entfernen Sie alle Messwertfilter aus der Filterleiste.",
```

Comment:
This is weird. How is this possible?

Reply:
That happens because last translations update was merged into branch _release_. Guys should rebase on develop  branch.

Reply:
Thanks.
This should be removed after rebased

Reply:
fixed